internal/cmd/asyncapi/command_export.go

```diff
@@ -155,9 +155,9 @@ func (c *command) export(cmd *cobra.Command, _ []string) (err error) {
 func (c *command) getChannelDetails(details *accountDetails, flags *flags) error {
 	output.Printf("Adding operation: %s\n", details.channelDetails.currentTopic.GetTopicName())
 	err := details.getSchemaDetails()
-	if err.Error() == "protobuf" {
+	if err == errors.New("protobuf is not supported") {
 		log.CliLogger.Info("Protobuf is not supported.")
-		return fmt.Errorf("protobuf")
+		return errors.New("protobuf is not supported")
```

Comment:
Don't you think that logging the error and returning the error is redundant?
```suggestion
		return err
```